repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py

```diff
@@ -0,0 +1,159 @@
+import json
+from collections import namedtuple
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.stdlib import api
+from leapp.models import InstalledRedHatSignedRPM, RpmTransactionTasks, RepositoriesSetupTasks
+
+
+REPOSITORIES_BLACKLIST = ('rhel8-buildroot', 'rhel8-crb')
+# FIXME: this mapping is not complete and will need to be manually updated frequently
+REPOSITORIES_MAPPING = {
```

Comment:
I personally think that this is the wrong place to map this TBH

Reply:
I agree. I think it was mentioned before, but the RHSM repositories IDs are handled by a separated file with the same distribution approach as PES data. Would make no sense add them hardcoded here. Such mapping can even happen outside of Leapp, as discussed in a Cabal.